### PR TITLE
style: refine tables

### PR DESCRIPTION
**Pull request by hkang1**
## Description

Fine tune styles relating to list tables.

## Commentary (optional)

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/src/components/ProgressBar.tsx [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
   const style = {
     backgroundColor: getStateColor(state),
     width: `${percent}%`,
   };
 
   return (
-    <div className={css.base} {...rest}>
-      <span className={css.progress} style={style} />
-    </div>
+    <Tooltip title={floatToPercent(percent / 100, 0)}>
```
Comment (hamidzr) [inline webui/react/src/components/ProgressBar.tsx:26]: :+1:

**Comment by hkang1** [comment]
> a lot of good changes thanks addressing all these. I noticed two visual issues:
> Screen Capture_select-area_20200803132905
> I'm glad we're adding the hover effect for rows! I have style comment: IMO changing the row background would look nicer.
> Screen Capture_select-area_20200803132828

Good call. I removed the shadow and applied the action blue border, similar to our interactive elements like the select filters, checkboxes, toggles, etc. I didn't do a background color because the multiselect applies a light blue background currently and didn't want to mess with that for now. Maybe revisit it again once migration is complete?

> Screen Capture_select-area_20200803132800

Great catch on this one 🤦  fixed

**Comment by hkang1** [comment]
I enlarged the target areas so when you click on the entire cell for multiselect or the task action dropdown, it will trigger the respective alternative actions.

File: webui/react/src/components/Table.tsx [hunk 2]
```diff
@@ -98,9 +97,16 @@ export const expermentDurationRenderer: ExperimentRenderer = (_, record) => {
...
+  return record.progress ? <ProgressBar
     percent={record.progress * 100}
-    state={record.state}
-    title={floatToPercent(record.progress, 0)} />;
+    state={record.state} /> : '-';
+};
+
+/* Table Helper Functions */
+
+export const isAlternativeAction = (event: React.MouseEvent): boolean => {
```
Comment (hamidzr) [inline webui/react/src/components/Table.tsx:107]: nit: could you add some comments on how /where people use this?

File: webui/react/src/components/Table.tsx [hunk 3]
```diff
@@ -98,9 +97,16 @@ export const expermentDurationRenderer: ExperimentRenderer = (_, record) => {
...
+  const target = event.target as unknown as HTMLElement;
```
Comment (hamidzr) [inline webui/react/src/components/Table.tsx:108]: I think this doesn't need to be cast to `unknown` first. removing that `as` should make it safer.

Comment (hkang1) [reply webui/react/src/components/Table.tsx:108]: ah you're right, nice!

File: webui/react/src/components/Link.tsx [hunk 4]
```diff
@@ -39,21 +39,23 @@ export const makeClickHandler = (
...
-      {props.children}
-    </span>;
-  }
+  const handleClick = useCallback(() => {
+    return makeClickHandler(path, onClick, popout);
+  }, [ path, onClick, popout ]);
 
-  return (
-    <a className={classes.join(' ')} href={path} rel={rel} onClick={handleClick}>
+  return props.disabled ? (
```
Comment (hamidzr) [inline webui/react/src/components/Link.tsx:51]: non-blocking: IMO the previous way of having two independent returns is more readable.

**Comment by hamidzr** [comment]
Screen Capture_select-area_20200805153136

LG. these two portions I think we could refine. left aligned dashes for progress seem a bit off (center align? although we don't center anything else) and then I think we could truncate the experiment descriptions but none of these are added in this PR.

**Pull request merged**